django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html

```diff
@@ -32,7 +32,7 @@
           <a class="related-widget-wrapper-link view-related" id="view_id_{{ name }}"
```

Comment:
^ Above – we also need to add the width and height for the "Delete" action link.

Reply:
I have added the width and height, though I got a bit stuck trying to test (trying to find out when the delete should appear).

By doing something hacky, this is the before:
![image](https://github.com/django/django/assets/77671865/c82d9d03-de0f-4f65-9745-5dc48703eace)

And this is the after:
![image](https://github.com/django/django/assets/77671865/f438bd4a-395a-4427-a109-bfecf5247f68)